### ISSUE TITLE
Grower module filter organization "not set" does not work

### ISSUE THREAD

**Issue by sebastiangaertner** [Greenstand/treetracker-admin-client#1044]
When trying to filter by grower that has no org assigned the filter does not work. See screenshot. 
![Screenshot 2023-03-15 at 07 54 58](https://user-images.githubusercontent.com/3283800/225210884-38da3849-2c81-44bd-a7aa-54ef681a808d.png)

**Comment by nmcharlton** [comment]
Duplicate of #869
Fixed deployed to dev and test environments.

**Comment by sebastiangaertner** [comment]
@nmcharlton was this ever released into production?

**Comment by nmcharlton** [comment]
> @nmcharlton was this ever released into production?

No. The production version is still using the legacy API which does not have the fix.
Getting the v2 Admin Panel into production has to be the priority for the Admin Panel, Microservices and Infrastructure teams.

**Comment by yasiru98** [comment]
@cynthiabondi, I'd like to work on this issue

**Comment by cynthiabondi** [comment]
done @yasiru98 :)
